.github/workflows/linux-qe-template.yml

```diff
@@ -16,11 +16,14 @@ on:
 
 jobs:
   linux-qe:
-    runs-on: [self-hosted, linux, testing-farm]
+    runs-on: ubuntu-24.04
     permissions:
       statuses: write # needed to update commit status (pending/failure/sucess)
       checks: write # as documented in https://github.com/mikepenz/action-junit-report?tab=readme-ov-file#pr-run-permissions
     steps:
+      - name: Check out repository code
```

Comment:
Why are you cloning the code here? to get the bunble versions? If that is the case it will not work on the PR for the new bundles version.

This is b/c this is a workflow_run which is executed on the context of the default branch of the upstream repo which has actual verisons not the ones you set on the PR....

For this to work you need to pass that information on an artifact (as we do with images...)

But as this is an edge case feel free to create a following issue with this may @albfan can take a look at it...as is a minor chnage but it requires to know about how this testing strategy works.

beyond this LGTM

Reply:
^^ @lilyLuLiu

Reply:
Thanks Adrian. Will fix this for a following pr.